apps/browser/lib/supabase/SupabaseWorkspace.ts

```diff
@@ -76,16 +76,16 @@ export class SupabaseWorkspace implements Workspace {
       throw new Error("Non-recursive rmdir is not supported");
     }
 
-    const path = this.toWorkspacePath(subpath);
+    const folderPath = path.resolve("/", subpath).slice(1);
```

Comment:
would like your eyes here @nerfZael since the `toWorkspacePath` concatenates the pre-defined chatId, I had to do this; because the UX we want to achieve is that the user may delete the chat (which is in the sidebar); and in order to have the `chatId` of the workspace, the chat needs to be opened

Reply:
This right now is using a SupabaseWorkspace of one ChatID (ex: 5) to delete another ChatID's workspace (ex: 6). To me a better route is to create a new SupabaseWorkspace use the ChatID you want to delete (6) and simply call `rmdir` with an empty `subpath`.